drivers/node/vsphere/vsphere.go

```diff
@@ -0,0 +1,248 @@
+package vsphere
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/portworx/sched-ops/task"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/node/ssh"
+	"github.com/sirupsen/logrus"
+	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/find"
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/govmomi/vim25/types"
+)
+
+const (
+	// DriverName is the name of the vsphere driver
+	DriverName = "vsphere"
+	// Protocol is the protocol used
+	Protocol = "https://"
+)
+
+const (
+	vsphereUname = "VSPHERE_USER"
+	vspherePwd   = "VSPHERE_PWD"
+	vsphereIP    = "VSPHERE_HOST_IP"
+)
+
+const (
+	// DefaultUsername is the default username used for ssh operations
+	DefaultUsername = "root"
```

Comment:
rename it to DefaultVsphereUsername as this is a username for vSphere

Reply:
Since this is already in the vsphere driver, I think we need to call it `DefaultVsphereUsername` same convention is being followed in the ssh driver